### DISCUSSION TITLE
LILYGO model:T3 v1.6.1 unable to connect to wifi

### DISCUSSION THREAD

**Discussion by xmttr1** [dl9rdz/rdz_ttgo_sonde#465] [Q&A]
I am trying to flash this LILYGO  V1.6.1 with the EPS32 FLASH DOWNLOAD TOOL V3.9.7. I am using dev20240913-C3-full.bin @ 0x1000. It accepts the load and the attempts searching for wifi. after a few seconds, it starts scanning for RS41. What tool can I use to configure this LILYGO. I tried to access 192.168.4.1 but fails

MODEL LORA32

config is:
SDA:21
SCL:22
RST: -1
LED:25
SPEC: -1
MAX: 20

**Comment by dl9rdz** [comment]
This all looks right. This board is the same as I am using here and should be well supported.

You should see an access point "RDZsonde" (password is also RDZsonde) and connect to that access point. After connecting, you should be able to access http://192.168.4.1.  Hard to tell without more details from your side what is not going right. Maybe two suggestions:
- Make sure it is http, not https
- If you are using an Android phone, make sure to turn of mobile data when accessing http://192.168.4.1
